db/helpers/collectorUtils.js

```diff
@@ -82,8 +82,24 @@ function validateVersion(version) {
   }
 }
 
+/**
+ * Find unassigned generators and assign them.
+ * @param {Object} seq - Sequelize object
+ * @returns {Promise} - Resolves to array of assigned generator db objects
+ */
+function assignUnassignedGenerators(seq) {
```

Comment:
Is it necessary to pass in the seq object here? Can't we just require it in this file?

Reply:
seq is here coming in as a parameter when we initialize model. We can probably get the seq instance from index file, but I noticed that we are usually passing in models for utils, probably because utils are used by corresponding models.
I am fine either ways, let me make the change.